simbad/command_line/__init__.py

```diff
@@ -339,7 +339,7 @@ def _simbad_contaminant_search(args):
     elif args.rot_program.lower() == "phaser":
         from simbad.rotsearch import PhaserRotationSearch
 
-        rotation_search = PhaserRotationSearch(temp_mtz, args.tmp_dir, args.mr_program,
+        rotation_search = PhaserRotationSearch(temp_mtz, args.mr_program, args.tmp_dir,
```

Comment:
That's what I was referring to in one comment ... it's confusing to mix arguments around. Keep the order the same